extensions/desktop/gnome/Makefile

```diff
@@ -17,13 +18,14 @@ clean:
 
 $(DEST_LAUNCHER):
 	@cat $(SRC_DIR)/init >> $(DEST_LAUNCHER)
-	# tail -n +2 to remove the shebang
+# tail -n +2 to remove the shebang
```

Comment:
Technically the tab is not a syntax error here. Though this is not the essential part.

Reply:
Oh, it is not, as it was working, just visual tolerance on my side